lib/react/rails/controller_renderer.rb

```diff
@@ -0,0 +1,17 @@
+class React::Rails::ControllerRenderer
+  include React::Rails::ViewHelper
+  include ActionView::Helpers::TagHelper
+  include ActionView::Helpers::TextHelper
+
+  attr_accessor :output_buffer
```

Comment:
Btw @garbles these few calls (`include ActionView::...` + `attr_accessor :output_buffer`), is this what's required to make your custom object work with view rendering?